opensrp-reveal/src/main/java/org/smartregister/reveal/adapter/OfflineMapAdapter.java

```diff
@@ -0,0 +1,54 @@
+package org.smartregister.reveal.adapter;
+
+import android.content.Context;
+import android.support.annotation.NonNull;
+import android.support.v7.widget.RecyclerView;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+
+import org.smartregister.reveal.R;
+import org.smartregister.reveal.model.OfflineMapModel;
+import org.smartregister.reveal.viewholder.AvailableOfflineMapViewHolder;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class OfflineMapAdapter extends RecyclerView.Adapter<AvailableOfflineMapViewHolder> {
+
+    private Context context;
+
+    private View.OnClickListener offlineMapClickHandler;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Perhaps 'offlineMapClickHandler' could be replaced by a local variable.](https://app.codacy.com/manual/OpenSRP/opensrp-client-reveal/pullRequest?prid=4217760)